versions.lock

```diff
@@ -1,23 +1,24 @@
 # Run ./gradlew --write-locks to regenerate this file
 com.atlassian.commonmark:commonmark:0.12.1 (1 constraints: 36052a3b)
-com.fasterxml.jackson.core:jackson-annotations:2.11.1 (10 constraints: b6b3f219)
-com.fasterxml.jackson.core:jackson-core:2.11.1 (15 constraints: 373c64bc)
-com.fasterxml.jackson.core:jackson-databind:2.11.1 (27 constraints: 6e00acf4)
-com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.11.1 (2 constraints: da3385ff)
-com.fasterxml.jackson.dataformat:jackson-dataformat-smile:2.11.1 (1 constraints: 7b1c94a4)
-com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.11.1 (3 constraints: 3225280e)
-com.fasterxml.jackson.datatype:jackson-datatype-guava:2.11.1 (2 constraints: 292b3aaf)
-com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.11.1 (6 constraints: f25e1c50)
-com.fasterxml.jackson.datatype:jackson-datatype-joda:2.11.1 (2 constraints: 292b3aaf)
-com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.11.1 (2 constraints: 292b3aaf)
-com.fasterxml.jackson.module:jackson-module-afterburner:2.11.1 (4 constraints: 33510b82)
-com.fasterxml.jackson.module:jackson-module-parameter-names:2.11.1 (2 constraints: b621beb6)
-com.github.ben-manes.caffeine:caffeine:2.8.8 (9 constraints: 019ca633)
+com.fasterxml.jackson:jackson-bom:2.12.1 (14 constraints: 18375425)
```

Comment:
@carterkozak haven't checked what bumped this (on mobile) but we ready for 2.12.x?

Reply:
Yep, this was intentional. The PR in CJR handled the new configuration, otherwise we risk folks upgrading to newer releases than we're prepared for which require slightly different setup.